### DISCUSSION TITLE
Excludes query terms from search results with a negation operator

### DISCUSSION THREAD

**Discussion by Boscop** [meilisearch/product#77] [Feedback & Feature Proposal]
With MeiliSearch, what's the equivalent of Google's negatíon operator? (`-foo`: exclude results that contain `foo`)
Is it doesn't exist yet, consider this a feature request :)

Btw, how can I invert the word position weighting, so that matches closer to the **end** of a document result in a higher score? (But it should only be inverted for one particular attribute, not all!)

**Comment by curquiza** [comment]
Hello @Boscop 

> With MeiliSearch, what's the equivalent of Google's negatíon operator? (-foo: exclude results that contain foo)
Is it doesn't exist yet, consider this a feature request :)

There is no equivalent for the moment, you can open a can in our roadmap if you want this feature -> https://roadmap.meilisearch.com/tabs/5-ideas

> Btw, how can I invert the word position weighting, so that matches closer to the end of a document result in a higher score? (But it should only be inverted for one particular attribute, not all!)

This is not possible at the moment, sorry.

**Comment by gmourier** [comment]
Hello @boscop! This issue has been turned into a feature request discussion on the product side. So people can now vote for it and discuss it here!

**Comment by sapristi** [comment]
Hello, has this been removed ? I think I saw it at some point, but now I can't find the discussion for this feature anywhere in https://roadmap.meilisearch.com/ 

Btw congratulations on v0.21 !

**Reply by gmourier** [reply]
Hi @sapristi 👋

It was never there from what I see on the dashboard! It's now settled. Thank you! 

https://roadmap.meilisearch.com/c/104-negative-keyword-search
